docs/azure/integrations-overview.md

```diff
@@ -113,9 +113,10 @@ The `RunAsExisting` method is used when a distributed application is executing i
 var builder = DistributedApplication.CreateBuilder();
 
 var existingServiceBusName = builder.AddParameter("existingServiceBusName");
+var existingServiceBusResourceGroup = builder.AddParameter("existingServiceBusResourceGroup");
```

Comment:
See late change in [[release/9.1] Require resource group on existing azure resources (dotnet/aspire#7734)](https://github.com/dotnet/aspire/pull/7734)